code/modules/media/jukebox.dm

```diff
@@ -405,20 +427,26 @@ var/global/loopModeNames=list(
 		attack_hand(user)
 
 /obj/machinery/media/jukebox/emag(mob/user)
-	current_song = 0
 	if(!emagged)
-		playlist_id = "emagged"
-		last_reload=world.time
-		playlist=null
-		loop_mode = JUKEMODE_SHUFFLE
-		emagged = 1
-		playing = 1
 		user.visible_message("<span class='warning'>[user.name] slides something into the [src.name]'s card-reader.</span>","<span class='warning'>You short out the [src.name].</span>")
-		update_icon()
-		update_music()
-		return 1
+		wires.CutWireIndex(JUKE_CONFIG)
 	return
 
+/obj/machinery/media/jukebox/proc/short()
```

Comment:
This makes emagging the jukebox pointless, as any fucking guy can do it. There's a reason it is the emagged playlist, and that is it is only accessible under this rare circumstance. And equally emagging it is meant to be just as destructive as it cannot be undone. This eliminates both of those considerations.